src/app/portlets/dot-edit-page/content/components/dot-edit-page-toolbar/dot-edit-page-toolbar.component.spec.ts

```diff
@@ -92,6 +94,10 @@ describe('DotEditPageToolbarComponent', () => {
                         provide: DotWorkflowService,
                         useClass: DotWorkflowServiceMock
                     },
+                    {
+                        provide: LoginService,
```

Comment:
All toolbar tests were broken now, because the action components needs more providers.